zrml/prediction-markets/src/lib.rs

```diff
@@ -1064,17 +1084,13 @@ mod pallet {
         /// The maximum allowed timepoint for the market period (timestamp or blocknumber).
         type MaxMarketPeriod: Get<u64>;
 
-        /// Shares of outcome assets and native currency
-        type AssetManager: ZeitgeistAssetManager<
-            Self::AccountId,
-            CurrencyId = Asset<MarketIdOf<Self>>,
-            ReserveIdentifier = [u8; 8],
-        >;
-
         /// The module identifier.
         #[pallet::constant]
         type PalletId: Get<PalletId>;
 
+        /// The origin that is allowed to approve / reject pending advised markets.
```

Comment:
Similar fix needs to be applied to the docs of `ApproveOrigin`.

```suggestion
        /// The origin that is allowed to reject pending advised markets.
```